connector-drawer/src/main/java/com/redhat/cloud/notifications/connector/drawer/DrawerPayloadBuilder.java

```diff
@@ -0,0 +1,82 @@
+package com.redhat.cloud.notifications.connector.drawer;
+
+import com.redhat.cloud.notifications.connector.drawer.constant.ExchangeProperty;
+import com.redhat.cloud.notifications.connector.drawer.model.DrawerEntry;
+import com.redhat.cloud.notifications.connector.drawer.model.DrawerEntryPayload;
+import com.redhat.cloud.notifications.connector.drawer.model.DrawerUser;
+import io.vertx.core.json.Json;
+import jakarta.enterprise.context.ApplicationScoped;
+import org.apache.camel.Exchange;
+import org.apache.camel.Message;
+import org.apache.camel.Processor;
+import org.apache.camel.util.json.JsonObject;
```

Comment:
This will break the imports order:
```suggestion
import io.vertx.core.json.JsonObject;
```